queries.sql

```diff
@@ -20,3 +20,34 @@ SELECT * FROM animals  WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
 BEGIN;
 UPDATE animals SET species = 'unspecified';
 ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '01/01/2022';
+SAVEPOINT delete_after;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO delete_after;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg  < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
```

Comment:
- [ ] Please include all queries from PR in to the `query.sql` file. You are currently missing following queries from `query.sql` file:

How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000? 

- [ ] The query for question `What is the average number of escape attempts per animal type of those born between 1990 and 2000? ` will only showing results for `pokemon` species  becuse of the condition `where speicies= 'pokemon'`. Please include the result for other species born between 1990 and 2000 as well by removing the `where speicies= 'pokemon'` condition from the query.